migrations/lib/common/set_store/simple_set.rb

```diff
@@ -22,7 +22,7 @@ def include?(value)
     end
 
     def bulk_add(records)
-      @store.merge(records)
+      records.each { |row| @store.add(row[0]) }
```

Comment:
`merge` is understandably more performant here, but the records from the DB are nested, leadind to a set of arrays if used directly. I could clean that up at the call site ((https://github.com/discourse/discourse/pull/33626/files#diff-76b20f97543c4d7b5f4e898939dcd73f8f787e862f03c944d5435cef1ead8716R14), but I want to test whether conditionally flattening records before merging introduces any performance penalties